x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -120,7 +120,7 @@ export const allowedExperimentalValues = Object.freeze({
   /**
    * Enables the Assistant Model Evaluation advanced setting and API endpoint, introduced in `8.11.0`.
    */
-  assistantModelEvaluation: false,
+  assistantModelEvaluation: true,
```

Comment:
Temporarily enabling model eval so we can test tracing in cloud. Please revert before merging!

```suggestion
  assistantModelEvaluation: false,
```